recipes/gmaps/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "gmaps" %}
+{% set version = "0.5.0" %}
+{% set sha256 = "e4d54540424de6d710097cddc2f29e38" %}
```

Comment:
You want to change this to '1fa461c7544fb95310b34e1a9be0d1e27af7c805f64a206eb560e9cdaef857da'

you can look these up at https://pypi.org/project/gmaps/#files